ui/app/AppLayouts/Wallet/adaptors/CollectiblesSelectionAdaptor.qml

```diff
@@ -140,6 +143,15 @@ QObject {
                 expression: root.enabledChainIds.includes(model.chainId)
                 expectedRoles: ["chainId"]
                 enabled: root.enabledChainIds.length
+            },
+            FastExpressionFilter {
```

Comment:
Pls rewrite it as `AnyOf { ValueFilter {}, ValueFilter{} }`, this is too slow :)